### PR TITLE
Exposing snakemake's --rerun-triggers (#170).

### PR DESCRIPTION
**Pull request by holtgrewe**

### CODE REVIEW

File: snappy_pipeline/apps/snappy_snake.py [hunk 1]
```diff
@@ -285,6 +288,17 @@ def main(argv=None):
             "not yet exist."
         ),
     )
+    group.add_argument(
+        "--rerun-triggers",
+        nargs="+",
+        choices=RERUN_TRIGGERS,
+        default=["mtime"],  # TODO: switch to RERUN_TRIGGERS,
+        help="Define what triggers the rerunning of a job. By default, "
```
Comment (eudesbarbosa) [inline snappy_pipeline/apps/snappy_snake.py:296]: The help info is misleading, we are doing exactly the opposite.

**Review by eudesbarbosa — COMMENTED** [review]
LGTM

File: snappy_pipeline/apps/snappy_snake.py (hunk 1 above)
Comment (holtgrewe) [reply snappy_pipeline/apps/snappy_snake.py:296]: Can we keep the fake help for now in your opinion?

Comment (eudesbarbosa) [reply snappy_pipeline/apps/snappy_snake.py:296]: Honestly, probably not. It would just cause unnecessary confusion.

Comment (holtgrewe) [reply snappy_pipeline/apps/snappy_snake.py]: OK fixed.

**Comment by holtgrewe** [comment]
@eudesbarbosa can you add an approving review?

**Review by eudesbarbosa — APPROVED** [review]

**Pull request merged**